sdk/servicebus/Microsoft.Azure.ServiceBus/src/Microsoft.Azure.ServiceBus.csproj

```diff
@@ -2,7 +2,7 @@
   <PropertyGroup>
     <AssemblyTitle>Azure ServiceBus SDK</AssemblyTitle>
     <Description>This is the next generation Azure Service Bus .NET Standard client library that focuses on queues &amp; topics. For more information about Service Bus, see https://azure.microsoft.com/en-us/services/service-bus/</Description>
-    <VersionPrefix>3.4.0</VersionPrefix>
+    <VersionPrefix>3.5.0</VersionPrefix>
```

Comment:
>3.5.0 [](start = 19, length = 5)

Major version needs to be changed here, and not minor version. But now to think of it, this should not be your responsibility. The person doing the release should take care of updating the version whenever he/she releases. Maybe you can revert this change. 
Sounds good @jsquire?

Reply:
I agree with both of Neeraj's points.   These changes are definitely worth a major bump due to some being breaking changes.  Better to only version during release, lest it keep getting bumped inadvertently.

Reply:
My preference would be to update the changelog.md as well as the version at the time when the decision to ship a new major version is made -- or we should figure out an approach to track the required version changes.

Reply:
The question that occurs to me with that approach how a contributor would know when they need to increment the version and when that has already been done prior?   I fear that we may have a gap with respect to "we're planning to release a new version and these changes are a part of it"  versus  "I'd like to include these changes for whenever a new major version is planned"  versus  "I'd like to include these priority changes and they should trigger a minor release and be done soon."

Reply:
FWIW, in the past (old repo) there were milestones and one of them was the next major. A PR/issue that would be a breaking change would be assigned to that milestone. When it comes to collaborators, especially external, during the PR work/review it becomes clear wherever a change is a breaking one or not and what version should it target (major, minor, or a patch).

Reply:
And, for consideration, it will be extremely helpful to see issues associated with the PRs that would be aggregated for each release, in the notes section. Similar to what was done here: https://github.com/Azure/azure-service-bus-dotnet/releases